api/MediaTrackConstraints.json

```diff
@@ -85,7 +85,7 @@
               "version_added": null
             },
             "webview_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
Here's the [commit that enabled this](https://chromium.googlesource.com/chromium/src/+/6093997931808df404b0350cfa4630a3a8bc6143) and a bunch of other things. It [landed in Chrome 59](https://storage.googleapis.com/chromium-find-releases-static/609.html#6093997931808df404b0350cfa4630a3a8bc6143).

Reply:
The majority of that stuff is not in BCD at all; looks like it may be from a different spec. I have not added it because that's outside the scope of this issue. I did update the version number for the stuff that clearly needed to be done though.

I think at this point it may make more sense to do a follow-up with any further Chrome updates, since Chrome clearly has some added stuff in these interfaces that come from other specs.